services/minespace-web/common/selectors/staticContentSelectors.js

```diff
@@ -96,10 +96,15 @@ export const getIncidentFollowupActionOptions = (state, isShowActiveOnly) =>
     isShowActiveOnly
   );
 
-// removes all expired compliance codes from the array
-export const getCurrentComplianceCodes = createSelector([getComplianceCodes], (codes) =>
-  codes.filter((code) => code.expiry_date === null || new Date(code.expiry_date) > new Date())
-);
+// marks all expired compliance codes as 'Repealed'
+export const getCurrentComplianceCodes = createSelector([getComplianceCodes], (codes) => {
+  return codes.map((code) => {
+    if (new Date(code?.expiry_date) < new Date()) {
+      code.description = `${code.description} (Repealed)`;
```

Comment:
Could we have added a boolean property so we're not reliant on substrings?

Reply:
I thought about that, and it's possible.  
The snag I found there that I didn't like was that the data is eventually being transformed into label/value options before being passed into the table, so they would no longer contain that boolean (originally I thought of just considering the `expiry_date` in the table to conditionally render the `(Repealed` text.